packages/dds/tree/src/simple-tree/objectNode.ts

```diff
@@ -200,6 +200,12 @@ function createProxyHandler(
 			setField(flexNode.getBoxed(fieldInfo.storedKey), fieldInfo.schema, value);
 			return true;
 		},
+		deleteProperty(target, viewKey): boolean {
```

Comment:
Might be worth adding a unit test for this scenario asserting the expected error message.